src/components/input/input-base.ts

```diff
@@ -107,6 +105,10 @@ export abstract class IgcInputBaseComponent extends FormAssociatedRequiredMixin(
 
   protected abstract renderInput(): TemplateResult;
 
+  protected renderValidatorContainer(): TemplateResult {
+    return IgcValidationContainerComponent.create(this);
```

Comment:
Hm, noticed doesn't specify an id like the combo and doesn't really use `aria-describedby` either. Somewhat assumed the re-slotting of `helper-text` from other components would at least solve this here, but seems like a nope. I know, more like a separate issue - feel free to log.